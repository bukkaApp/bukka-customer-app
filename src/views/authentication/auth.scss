.auth-page {
    @media (max-width: 576px) {
       background-color: white!important;
    }
    @media (max-width: 767px) {
       min-height: calc(#{"100vh - 65px"});
    }
}

.auth-wrapper {
    @media (max-width: 576px) {
        box-shadow: none!important;
     }
}
