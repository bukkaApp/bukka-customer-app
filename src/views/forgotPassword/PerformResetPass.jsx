import React from 'react';
import RetrieveSection from './components/RetrieveSection';

const PerformResetPass = props => <RetrieveSection {...props} />;

export default PerformResetPass;
