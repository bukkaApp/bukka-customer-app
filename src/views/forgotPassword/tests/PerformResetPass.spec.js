import React from 'react';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import { MemoryRouter } from 'react-router-dom';
import PerformResetPass from '../PerformResetPass';
import auth from './__mocks__/changePasswordAction.json';


const initialState = {
  navbarAuthReducer: { type: '/login' },
  validateTokenReducer: { isValidUser: true, requested: true, },
  changePasswordReducer: { requested: true, errorMessage: 'error occur' },
  authenticationReducer: {
    user: {},
    status: {
      authenticated: false,
      error: false
    },
    errorMessage: ''
  }
};

const middlewares = [];
const mockStore = configureStore(middlewares);
const store = mockStore(initialState);


describe('Perform Reset Pass component', () => {
  const props = {
    status: {
      authenticated: false,
      error: false
    },
    history: { push: jest.fn() },
    location: { search: auth.token }
  };
  const wrapper = mount(
    <Provider store={store}>
      <MemoryRouter>
        <PerformResetPass {...props} />
      </MemoryRouter>
    </Provider>
  );

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
