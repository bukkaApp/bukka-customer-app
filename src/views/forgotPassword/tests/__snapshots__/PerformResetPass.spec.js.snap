// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Perform Reset Pass component renders properly 1`] = `ReactWrapper {}`;
